qemu/tests/migration_with_file_transfer.py

```diff
@@ -55,8 +57,15 @@ def run_and_migrate(bg):
                             vm.params["vhost"] = "vhost=off"
                         elif vhost_status == "vhost=off":
                             vm.params["vhost"] = "vhost=on"
+                    migration_exec_cmd_src = params.get("migration_exec_cmd_src")
+                    migration_exec_cmd_dst = params.get("migration_exec_cmd_dst")
+                    if mig_protocol == "exec":
+                        migration_exec_cmd_src %= mig_file
+                        migration_exec_cmd_dst %= mig_file
                     vm.migrate(mig_timeout, mig_protocol, mig_cancel_delay,
```

Comment:
I think I missed `@default_exec` variant, it doesn't have exec_cmd, so should skip it.

Reply:
Due to src and dst cmd both appear in pairs, so I think judge only one of the parameters. Please correct me if I'm wrong.